constants/misc.js

```diff
@@ -1,8 +1,8 @@
 import createConstants from 'lib/utils/createConstants';
 
-import TOPICS from 'constants';
+import { TOPICS } from 'constants';
 
-export const TOPIC = createConstants(TOPICS);
+export const TOPIC = createConstants(...TOPICS);
```

Comment:
```suggestion
export const TOPIC = createConstants(TOPICS);
```

Reply:
I remember fixing this, there was a bug causing TOPICS not to be exported at all. Why do you want to get rid of `...`?

Reply:
seems to be redundant